src/lib/y2packager/widgets/simple_language_selection.rb

```diff
@@ -95,7 +96,7 @@ def items
         @items = languages.map do |lang|
           [lang, LanguageTag.new(lang).name(lang_map_cache: lmap)]
         end
-        @items.reject! { |_lang, name| name.nil? }
+        @items.compact!
```

Comment:
I'm not sure if that works the same way as before:
```
irb(main):001:0> [ ["foo", nil] ].compact
=> [["foo", nil]]
```

Reply:
yeah, here it is bug. As it expects it is hash. For hash it works, for array of arrays not.